pkg/timestream/executor_test.go

```diff
@@ -99,95 +102,110 @@ func TestGenerateTestData(t *testing.T) {
 		from ` + table + `
 		where time > ago(30m)
 			AND (measure_name = 'cpu_user' or measure_name = 'cpu_system')
-			and availability_zone ='us-east-1-1' 
+			and availability_zone ='us-east-1-1'
 			and microservice_name = 'hercules'
-			and region = 'us-east-1' 
-			and cell = 'us-east-1-cell-1' 
+			and region = 'us-east-1'
+			and cell = 'us-east-1-cell-1'
 			and silo = 'us-east-1-cell-1-silo-1'
 			and instance_type = 'r5.4xlarge'
 			group by measure_name, availability_zone, region, cell, silo, instance_type, instance_name
 		`,
 	}
 
 	m["some-timeseries.json"] = models.QueryModel{
-		RawQuery: `SELECT 
-			region, 
-			cell, 
-			silo, 
-			availability_zone, 
+		RawQuery: `SELECT
+			region,
+			cell,
+			silo,
+			availability_zone,
 			microservice_name,
-			instance_name, 
-			process_name, 
+			instance_name,
+			process_name,
 			jdk_version,
 			CREATE_TIME_SERIES(time, measure_value::double) AS gc_reclaimed
 		FROM ` + table + `
 		WHERE time > ago(1h)
 			AND measure_name = 'gc_reclaimed'
-			AND region = 'ap-northeast-1' 
-			AND cell = 'ap-northeast-1-cell-5' 
+			AND region = 'ap-northeast-1'
+			AND cell = 'ap-northeast-1-cell-5'
 			AND silo = 'ap-northeast-1-cell-5-silo-2'
-			AND availability_zone = 'ap-northeast-1-3' 
+			AND availability_zone = 'ap-northeast-1-3'
 			AND microservice_name = 'zeus'
-		GROUP BY region, 
-			cell, 
-			silo, 
-			availability_zone, 
+		GROUP BY region,
+			cell,
+			silo,
+			availability_zone,
 			microservice_name,
-			instance_name, 
-			process_name, 
-			jdk_version		
+			instance_name,
+			process_name,
+			jdk_version
 		ORDER BY AVG(measure_value::double) DESC
 		LIMIT 10`,
 	}
 
-	for key, value := range m {
-		writeTestData(key, value, t)
-	}
-}
-
-// This will write the results to local json file
-func writeTestData(filename string, query models.QueryModel, t *testing.T) {
```

Comment:
I had to remove this function since the linter was complaining that it was not being used :shrug: but it was only called in one place